internal/platform/mremap_other.go

```diff
@@ -12,21 +12,6 @@ func remapCodeSegmentAMD64(code []byte, size int) ([]byte, error) {
 	return b, nil
 }
 
-// mustMunmapCodeSegment panics instead of returning an error to the
-// application.
-//
-// # Why panic?
-//
-// It is less disruptive to the application to leak the previous block if it
-// could be unmapped than to leak the new block and return an error.
-// Realistically, either scenarios are pretty hard to debug, so we panic. There
-// is less as the dominant runtime.GOOS (linux) doesn't use this anyway.
-func mustMunmapCodeSegment(code []byte) {
-	if err := munmapCodeSegment(code); err != nil {
-		panic(err)
-	}
-}
-
```

Comment:
Moved to `internal/platform/platform.go` because it is used in `mremap_linux.go` now when huge pages are enabled. I also edited out the last line of the godoc.